src/lib/severity.js

```diff
@@ -1,4 +1,7 @@
 'use strict';
+
+const EmptyRegistry = require('./registry/empty-registry');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/loggin-js/loggin-js/pullRequest?prid=7629800)
```suggestion
const EmptyRegistry = require("./registry/empty-registry");
```